corehq/apps/sms/forms.py

```diff
@@ -1001,6 +1005,12 @@ def clean_name(self):
 
         return value
 
+    def clean_display_name(self):
+        value = self.cleaned_data.get("display_name")
+        if re.compile(r"\d").search(value) is not None:
+            raise ValidationError(_("Display name may not contain any numbers."))
```

Comment:
I'm curious about this requirement. The validation logic will allow any character other than numeric digits. Does that mean special/punctuation characters are allowed? If yes, why not numbers? Or maybe it should only allow ASCII characters?

Reply:
Good point. Given that this is an internal tool I'm tempted to say no validation is even necessary. I don't have a strong opinion on this though so if only allowing ASCII characters seems better I'll make that change.

Reply:
> Given that this is an internal tool I'm tempted to say no validation is even necessary.

I'm pro dropping the validation. This seems like the kind of thing someone might read later and say "Why was it done that way?" A comment explaining the "why" might be nice if the validation is kept.

Reply:
Removed validation.